Check for leap year

```diff
@@ -0,0 +1,28 @@
+# Python program to check if year is a leap year or not
+
+year = 2000
+
+# To get year (integer input) from the user
+
+# year = int(input("Enter a year: "))
+
+if (year % 4) == 0:
+
+   if (year % 100) == 0:
+
+       if (year % 400) == 0:
+
+           print("{0} is a leap year".format(year))
+
+       else:
+
+           print("{0} is not a leap year".format(year))
+
+   else:
+
+       print("{0} is a leap year".format(year))
+
+else:
+
+   print("{0} is not a leap year".format(year))
```

Comment:
Remove unnecessary spaces between lines.
Needless nested `if` statments. Could be combined into a single `if` .

- - - 

Such algorithem already exists in the `calendar`  library.
[docs.python.org/3/library/calendar.html](https://docs.python.org/3/library/calendar.html)
The code will look as such:
`print('is a leap year') if calendar.isleap(year) else print('not a leap year')`